eq-author/src/constants/validationMessages.js

```diff
@@ -75,6 +75,18 @@ export const rightSideErrors = {
   },
 };
 
+export const destinationErrors = {
+  ERR_DESTINATION_MOVED: {
```

Comment:
constants

Reply:
I'm not sure if this needs to be as this is the constants file for the error messages.